### ISSUE TITLE
Setup debug locales to match addons-frontend

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-server#2817]
On addons-frontend we have 2 debug languages/locales:
- `dbl` which is debug `ltr` (left to right) 
- `dbr` which is debug `rtl` (right to left)

Both are generated with podebug. The `rtl` one used `flipped` and the `ltr` one uses `unicode`

It would be nice if we could do the same in Olympia or otherwise sync up the way we do debug locales for both projects.

**Comment by EnTeQuAk** [comment]
Assigning this to me, I'll take a look at it while working on better l10n automation.

**Comment by EnTeQuAk** [comment]
This isn't solved yet, it's initial implementation got reverted and maybe I'll just wait till @muffinresearch finishes his library to properly generate the RTL locales. The original PR can then be re-used.

**Comment by muffinresearch** [comment]
@EnTeQuAk the lib is released on npm see https://github.com/mozilla/potools/ for details.

**Comment by ValentinaPC** [comment]
Verified this on AMO-dev FF53(Win 7)
- **`dbg`** which is debug ltr (left to right) - returns an `Oops...` page
- **`dbr`** which is debug rtl (right to left) - returns an `502 Bad Gateway` page
Screenshots:
![dbg](https://cloud.githubusercontent.com/assets/15685960/26314678/67735040-3f17-11e7-94f2-cf665e69a527.png)
![dbr](https://cloud.githubusercontent.com/assets/15685960/26314677/6772db38-3f17-11e7-82fa-0705d5c1cef2.png)

Reopening.

**Comment by EnTeQuAk** [comment]
Hmm, that's weird. I'll take a look at this early tomorrow and see
what's wrong, hoping that dev isn't completely unusable.

On Mon, May 22, 2017, at 09:52 PM, ValentinaPC wrote:
> Reopened #2817[1].


> — You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub[2], or mute the
> thread[3].> 

Links:

  1. https://github.com/mozilla/addons-server/issues/2817
  2. https://github.com/mozilla/addons-server/issues/2817#event-1091965872

**Comment by ValentinaPC** [comment]
> hoping that dev isn't completely unusable.

Beside these 2 pages, dev seems to work well. :)

**Comment by EnTeQuAk** [comment]
I know I fixed this before, this is the error: https://sentry.prod.mozaws.net/operations/olympia-dev/issues/608345/ - I'll open a new issue and fix this once I'm in the office in a few :)

**Comment by EnTeQuAk** [comment]
Will be handled inhttps://github.com/mozilla/addons-server/pull/5471

**Comment by ValentinaPC** [comment]
@EnTeQuAk : This issue was not fixed by https://github.com/mozilla/addons-server/issues/5470, which is closed.
The "Oops..." and "502 Bad Gateway" errors are still displayed.

**Comment by EnTeQuAk** [comment]
Yeah, unfortunately -dev isn't updating at the moment. I'll update the issue once it actually lands.

**Comment by ValentinaPC** [comment]
Verified dbl si dbr "languages" in AMO-dev FF53(Win 7).
Both options are loading the corresponding pages.
Postfix screenshots:
![dbl](https://cloud.githubusercontent.com/assets/15685960/26400998/5f13bb80-408c-11e7-9bc6-0b6f2d9c9249.png)
![dbr](https://cloud.githubusercontent.com/assets/15685960/26400997/5f017010-408c-11e7-8d37-ec4ac737eae1.png)

**Comment by vcarciu** [comment]
@EnTeQuAk : 
Can we solve this on Stage too? Both Debug Robot languages are returning  502 Bad Gateway

**Comment by EnTeQuAk** [comment]
interesting, this appears to 502 on -dev now for me too. I'll investigate.
